ebuild-writing/users-and-groups/text.xml

```diff
@@ -103,6 +103,175 @@ The defaults are the user and primary group for the owner, and 0755 for the
 permissions.
 </p>
 
+<important>
+Whenever possible, the default shell and home directory should be
+used. The rationale for this is explained below.
+</important>
+
+<p>
+You should embark upon a GLEP81 migration like an EAPI
+update. Rather than simply copy your user's settings into an
+<c>acct-user</c> package, you should take the opportunity to
+re-evaluate your user's name, shell, home directory, and its
+permissions. Our GLEP 81 implementation will reveal many user
+management issues that were allowed to fester in the past.
+</p>
+
+<subsection>
+<title>Choosing a shell</title>
+<body>
+<p>
+In most cases, the default shell (that is, no shell) should be
+used. Services can still be started as a user who has no shell, and
```

Comment:
```suggestion
used. Services can still be started as a user that has no shell, and
```

I'm almost tempted to say that you were the person who quoted either "The Elements of Style" or Knuth about using `that` vs `who` :)

Alternatively, this would also work I think:

```suggestion
used. Services can still be started as a user with no shell, and
```